### PR TITLE
[bitnami/milvus] Fix milvus config

### PR DESCRIPTION
**Pull request by rafariossaa**
### Description of the change

Milvus 2.3.x needs a config file for the log

### Applicable issues

- fixes #

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/milvus/templates/_helpers.tpl
Comment (javsalgar) [inline bitnami/milvus/templates/_helpers.tpl:971]: Shouldn't this be copied to the log-config volume?

Comment (rafariossaa) [reply bitnami/milvus/templates/_helpers.tpl:971]: The `log-config` volume is a configmap where the `glog.conf` is provided. Hence we need to copy it to `/opt/bitnami/milvus/configs`, but this done via `prepare-mivlus` init container. 

The init container `prepare-milvus` mounts the volume `rendered-config` on ` /bitnami/milvus/rendered-conf/`, this same volume is mounted later by the `milvus` container on `/opt/bitnami/milvus/configs`.

File: bitnami/milvus/templates/_helpers.tpl
Comment (migruiz4) [inline bitnami/milvus/templates/_helpers.tpl:417]:
```suggestion
{{- printf "%s-log" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" -}}
```

**Review by migruiz4 — COMMENTED** [review]
Just a minor suggestion

File: bitnami/milvus/Chart.yaml
Comment (migruiz4) [inline bitnami/milvus/Chart.yaml:51]: I have just merged a milvus update 5.1.0, please update this before merging

**Review by javsalgar — APPROVED** [review]

**Review by migruiz4 — APPROVED** [review]

**Comment by Uijeong97** [comment]
@rafariossaa @migruiz4 @javsalgar 

Is this PR still under review? 
I'm trying to use version 2.3.x, but I don't have the glog.conf file mentioned in the above PR. 
Would it be possible for you to review and merge the PR?

**Review by Uijeong97 — APPROVED** [review]

**Review by Uijeong97 — COMMENTED** [review]
I tried it on 2.3.3 and it deployed well 😆

**Comment by rafariossaa** [comment]
Hi,
  Yes, we are waiting for next release of milvus, as we are currently experiencing some issues in our CI/CD when building the current version.

**Pull request closed without merging**
